app-backend/build.gradle.kts

```diff
@@ -22,24 +22,32 @@ repositories {
 }
 
 kapt {
-    correctErrorTypes = true
+    correctErrorTypes= true
 }
 
 dependencies {
     implementation("org.springframework.boot:spring-boot-starter-web")
-    implementation("com.fasterxml.jackson.module:jackson-module-kotlin")
-    implementation("org.springframework.boot:spring-boot-starter-data-jpa")
     implementation("org.springframework.boot:spring-boot-starter-logging")
+    implementation("org.springframework.boot:spring-boot-starter-data-jpa")
+    implementation("org.springframework.boot:spring-boot-starter-security")
+    implementation("org.springframework.boot:spring-boot-starter-jdbc")
+    implementation("org.springframework.session:spring-session-jdbc")
```

Comment:
Let's delete this dependency because
1) it's redundant 
2) Exceptions will sometimes occur with this dependency because the spring_session table does not exist